flows/vision-zero/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:22.04
+ENV TZ="America/Chicago"
+RUN apt update
+RUN apt-get update && \
+    apt-get install -yq tzdata && \
+    ln -fs /usr/share/zoneinfo/America/Chicago /etc/localtime && \
+    dpkg-reconfigure -f noninteractive tzdata
+RUN apt upgrade -y
+RUN apt install -y vim magic-wormhole aptitude python3-pip docker.io rsync p7zip-full black
+RUN pip install prefect
+RUN pip install sysrsync GitPython boto3
```

Comment:
@frankhereford can these `pip install` commands be combined into a single `requirements.txt` file (with pinned version numbers), and installed here with ` RUN pip install -r requirements.txt`?

Reply:
@johnclary wrote: 
> @frankhereford can these pip install commands be combined into a single requirements.txt file (with pinned version numbers), and installed here with RUN pip install -r requirements.txt?

Broadly yes, I think it's more pythonic to utilize pip's ability to freeze out an environment and install from the requirements.txt which is generated. However, it's a little bit tricky here because the Dockerfile is defining the environment for the agent which can run multiple flows, and I envision each flow having a requirements.txt. I will investigate a way to have the Dockerfile iterate over each flow's requirements.txt and install from there. 

I'm not sure how much this matters, but one thing the docker file's `pip` invocations are doing is installing these as system-wide available python libraries within the dockerized linux host and not into a encapsulated `venv`. 

I am going to spin off an issue to address this, and I hope maybe you can I can sync on what I come up with or what I plan to come up with, depending on how it goes.

Reply:
OK, I think I misunderstood. I thought the per-agent Dockerfile was meant to provide an environment for all flows that this agent would run. As you add more flows to this agent's scope (e.g. the narrative extractor), do those requirements get stuffed somewhere else?

FWIW if it makes life easier I think it's fine to let one requirements.txt grow to support all of an agent's tasks. If an agent is generally managing roughly a repo's worth of ETLs, it would hopefully be rare for us to run into major dependency conflicts.